firefox-ios/Tests/XCUITests/BaseTestCase.swift

```diff
@@ -284,6 +284,23 @@ class BaseTestCase: XCTestCase {
     func waitForTabsButton() {
         mozWaitForElementToExist(app.buttons[AccessibilityIdentifiers.Toolbar.tabsButton], timeout: 15)
     }
+
+    func enableHiddenFeature(feature: String) {
```

Comment:
I'm not sure if I should ask the developers to add a command line argument. Since this feature is listed under the debug menu, let me access the feature from here. This code snippet may be useful in the future for testing something under the debug menu.

Reply:
This is a great way to enable features in the developer menu that we could use in the future. However, I don't think this is needed here... I can see the Firefox Suggest menu in main, it is not available from homescreen the very first time but once you open a site, you can see it.

Reply:
I may be missing something here but I don't think this is needed. It is adding a lot of complexity to the test. Suggest is enabled by default in main:
![Simulator Screenshot - iPhone 15 Pro - 2023-11-17 at 11 30 12](https://github.com/mozilla-mobile/firefox-ios/assets/1897507/78f97105-be49-481d-aee1-4c65a60f525c)

And just by going to a second page, the menu is shown.

```
func testFirefoxSuggestExists() {
        navigator.openURL("example.com")
        navigator.performAction(Action.OpenNewTabFromTabTray)
        navigator.goto(URLBarOpen)
        app.textFields["address"].typeText("sho")
        mozWaitForElementToExist(app.tables["SiteTable"]

```